integration-tests/schemas/scaffolding/api/apiWrapper.js

```diff
@@ -0,0 +1,72 @@
+module.exports = function ApiWrapper(polkadotApi, signerAccountKeys) {
```

Comment:
Not sure how I feel about having an API wrapper and then needing tests for it as well as part of the scaffolding.

Reply:
I think that if our goal is to thoroughly test e2e blockchain workflows, we will need a layer that encapsulates the boilerplate for recording asynchronous side effects. In theory, we could stuff that boilerplate into each test, but I felt that having a layer of indirection that contained all of that boilerplate would make writing the tests much easier (and fwiw, for these tests it did).

Thoughts?